src/Evolution/Systems/ScalarAdvection/TimeDerivativeTerms.cpp

```diff
@@ -15,9 +15,12 @@ namespace ScalarAdvection {
 
 template <size_t Dim>
 void TimeDerivativeTerms<Dim>::apply(
-    gsl::not_null<Scalar<DataVector>*> /*non_flux_terms_dt_vars*/,
-    gsl::not_null<tnsr::I<DataVector, Dim>*> flux, const Scalar<DataVector>& u,
+    const gsl::not_null<Scalar<DataVector>*> /*non_flux_terms_dt_vars*/,
+    const gsl::not_null<tnsr::I<DataVector, Dim>*> returned_velocity_field,
```

Comment:
Please rename this to `temp_velocity_field`, this is how we've been naming the returned temporaries (when the name matches an input) in the other systems, so it would be nice to keep it consistent here.